### ISSUE TITLE
Ability to choose version of client build

### ISSUE THREAD

**Issue by andrewheberle** [rustdesk/rustdesk-server-pro#298]
We have some older Ubuntu installs that are due for retirement, but for the moment we would like to stop supporting them via TeamViewer.

Unfortunately the current version of RustDesk built using the Pro version custom client builder does not work, so I wondered if it's possible to choose to build an older version of the client to support these legacy systems?

I would think this should come with a warning that older versions may have bugs or security problems, but the ability to choose the specific version would be good to have.

Also having the custom build show the version of the client that has been built would be very helpful too.

**Comment by rustdesk** [comment]
Ubuntu 18?

**Comment by andrewheberle** [comment]
Yes (unfortunately)

**Comment by rustdesk** [comment]
Our dev is supposed to run on 18. @fufesou follow it up.

**Comment by fufesou** [comment]
@andrewheberle Hi, what error are you facing?

You may need to add a source for `gstreamer1.0-pipewire`.

https://github.com/rustdesk/rustdesk/discussions/6148#discussioncomment-9295883

I also find an error in the service. Ubuntu 18 requires absolute path in the service file. I'll fix it.

```
ExecStop=pkill -f "rustdesk --"
```

**Comment by andrewheberle** [comment]
Let me re-check the problem tomorrow as this may be a local problem.

But also good you squashed that bug in the systemd service unit.

**Comment by andrewheberle** [comment]
@rustdesk I'm very sorry...the Ubuntu version was actually 16 LTS on this install.

Once we upgraded to Ubuntu 18.04 LTS the custom client installed ok, however the bug fixed in https://github.com/rustdesk/rustdesk/pull/8680 did mean the install didn't complete properly, but for the moment I've manually updated the service unit.

Sorry for the confusion I should have done more checking before lodging the issue.
